main.go

```diff
@@ -120,7 +120,8 @@ func main() {
 		BuildModule{Database: db, MessageQueue: mq},
 		TokenModule{Database: db},
 		BranchModule{Database: db},
-		ProviderModule{Database: db}}
+		ProviderModule{Database: db},
+		HealthModule{}}
```

Comment:
These are still not anonymous. Suggest moving the deprecated registration to before the basic auth as well

![Screenshot from 2021-08-23 09-13-09](https://user-images.githubusercontent.com/2477952/130405768-fd7fb2c9-77ae-4301-8c7d-77207c8a29cd.png)